fenix/app/src/androidTest/java/org/mozilla/fenix/experimentintegration/conftest.py

```diff
@@ -0,0 +1,132 @@
+import io
+import json
+import os
+import subprocess
+import time
+
+import pytest
+import requests
+
+from experimentintegration.gradlewbuild import GradlewBuild
+
+KLAATU_SERVER_URL = "http://localhost:1378"
+KLAATU_LOCAL_SERVER_URL = "http://localhost:1378"
+
+here = os.path.dirname(__file__)
+
+
+def load_branches():
+    branches = []
+    data = requests.get(f"{KLAATU_SERVER_URL}/experiment").json()
+    for item in data:
+        if isinstance(item, dict):
+            continue
+        else:
+            data = item
+    experiment = requests.get(data).json()
+    for item in experiment["branches"]:
+        branches.append(item["slug"])
+    return branches
+
+
+@pytest.fixture
+def gradlewbuild_log(pytestconfig, tmpdir):
+    gradlewbuild_log = f"{tmpdir.join('gradlewbuild.log')}"
+    pytestconfig._gradlewbuild_log = gradlewbuild_log
+    yield gradlewbuild_log
+
+
+@pytest.fixture
+def gradlewbuild(gradlewbuild_log):
+    yield GradlewBuild(gradlewbuild_log)
+
+
+@pytest.fixture(name="experiment_data")
+def fixture_experiment_data(experiment_url):
+    data = requests.get(experiment_url).json()
+    for item in data["branches"][0]["feature"]["value"]["messages"].values():
```

Comment:
A few times in this module the json responses are indexed. I wonder if there is value in dumping the json into data classes or using [match-cases](https://docs.python.org/3/whatsnew/3.10.html#pep-634-structural-pattern-matching)?

Reply:
Match Cases are cool! For dumping the data... not sure. I think defining a dataclass would add some complexity I don't think is needed.